internal/pkg/cli/create.go

```diff
@@ -36,7 +36,8 @@ Tip:
 const createBranchLongDesc = `Command "create branch"
 
 - Creates a new dev branch in the project remote state.
-- Local changes have no effect, so it is recommended to first use the "push" command.
+- The new branch will be a copy of the current remote state of the main branch.
+- It is recommended to first "push" local changes in the main branch, if any.
```

Comment:
Better help.

Reply:
Ta čárka před if by tam být neměla. (Já s těma čárkama v angličtině bojuju pořád, tak jsem si to dal raději do Grammarly. ​🙂
![CleanShot 2021-11-15 at 10 53 02](https://user-images.githubusercontent.com/215660/141760623-e1562ca1-3714-4a79-87fb-98c3508d0e36.jpg)

Reply:
:+1:

Reply:
vdaka, chcem exportovat vsetky texty (z funkcionalnych testov) a dat Hanke nech to skontroluje, ale este som sa k tomu nedostal.